### PR TITLE
docs: update the project participants' list

### PR DESCRIPTION
**Pull request by beryl678**
## What's Changed in this PR

*Describe the change in this PR*

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

## Summary by CodeRabbit

- **Documentation**
  - Updated `CONTRIBUTING.md` to include a section listing individual committers, all contributors, and team members with their GitHub handles and links.

### CODE REVIEW

**Review by killme2008 — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -72,3 +72,244 @@ Then start a local server with `pnpm run dev`.
...
+
+[CookiePieWw](https://github.com/CookiePieWw)
+
+[etolbakov](https://github.com/etolbakov)
+
+[irenjj](https://github.com/irenjj)
+
+[KKould](https://github.com/KKould)
+
+[NiwakaDev](https://github.com/NiwakaDev)
```
Comment (tisonkun) [inline CONTRIBUTING.md:88]:
```suggestion
* [CookiePieWw](https://github.com/CookiePieWw)
* [etolbakov](https://github.com/etolbakov)
* [irenjj](https://github.com/irenjj)
* [KKould](https://github.com/KKould)
* [NiwakaDev](https://github.com/NiwakaDev)
```

Maybe we can use an unordered list. Ditto other sections.

**Comment by tisonkun** [comment]
I'll send a patch against the main repo, not the docs repo.

**Comment by tisonkun** [comment]
Here it is.

**Pull request closed without merging**